### PR TITLE
Service Fabric 8.0 Release Notes

### PR DESCRIPTION
**Pull request by sukanyamsft**

### CODE REVIEW

File: release_notes/Service_Fabric_ReleaseNotes_80.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+Microsoft Azure Service Fabric 8.0 Release Notes
+
+* [Key Announcements](#key-announcements)
+* [Current And Upcoming Breaking Changes](#Current-And-Upcoming-Breaking-Changes)
+* [Service Fabric Runtime](#service-fabric-runtime)
+* [Service Fabric Common Bug Fixes](#service-fabric-common-bug-fixes)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Key Annoucements
```
Comment (peterpogorski) [inline release_notes/Service_Fabric_ReleaseNotes_80.md:27]: We should make sure we call out .Net 5 support as GA as one of the top items.

File: release_notes/Service_Fabric_ReleaseNotes_80.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+* Ability to move stateless service instances
+* Ability to add parameterized DefaultLoad in the application manifest
+* For singleton replica upgrades - ability to have some of the cluster level settings to be defined at an application level
+* Ability for smart placement based on node tags
+* Ability to define percentage threshold of unhealthy nodes that influence cluster health
+* Ability to query top loaded services
+* Ability to add a new interval for new error codes 
+* Capability to mark service instance as completed
+* Enable support for .NET 5 for Windows
+* Added readiness probe for containerized applications
```
Comment (peterpogorski) [inline release_notes/Service_Fabric_ReleaseNotes_80.md:37]: Is there also a liveness probe or just readiness?

File: release_notes/Service_Fabric_ReleaseNotes_80.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+* The simplest mitigation, when possible, is to delete and recreate the application in SF 7.0.<br/>
+* The other option, is to upgrade the application in SF 7.0 (for example, with a version only change).<br/>
+
+If the application is stuck in rollback, the rollback has to be first completed before the application can be upgraded again.
+
+
+
+## Service Fabric Runtime
+
+| Versions | IssueType | Description | Resolution |
```
Comment (peterpogorski) [inline release_notes/Service_Fabric_ReleaseNotes_80.md:63]: We should also include support for the wave based upgrade model. You can find the documentation here: https://github.com/MicrosoftDocs/azure-docs-pr/pull/151266

File: release_notes/Service_Fabric_ReleaseNotes_80.md (hunk 1 above)
Comment (sukanyamsft) [reply release_notes/Service_Fabric_ReleaseNotes_80.md:27]: Done.

File: release_notes/Service_Fabric_ReleaseNotes_80.md (hunk 2 above)
Comment (sukanyamsft) [reply release_notes/Service_Fabric_ReleaseNotes_80.md:37]: Just readiness.

File: release_notes/Service_Fabric_ReleaseNotes_80.md (hunk 3 above)
Comment (sukanyamsft) [reply release_notes/Service_Fabric_ReleaseNotes_80.md:63]: Added. Please review.

**Review by peterpogorski — APPROVED** [review]

**Pull request merged**